src/components/Content/Event.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import EventModel from '../../endpoints/models/EventModel'
+import RemoteContent from './RemoteContent'
+
+import style from './Event.css'
+import EventPlaceholder1 from '../../components/Content/assets/EventPlaceholder1.jpg'
```

Comment:
There imports can be simplified to:

`'./assets/EventPlaceholder1.jpg'`

Reply:
copy n paste :D